### PR TITLE
Added CONTRIBUTING.md file for documentation

### PR DESCRIPTION
**Pull request by ohmschrodinger**
issue no 2 [https://github.com/ajaynegi45/upchaar-aayog/issues/2](url)
@ajaynegi45

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+   ```
+4. Push to your branch:
+   ```bash
+   git push origin feature/YourFeature
+   ```
+5. Submit a pull request
+
+---
+
+### Instructions for Using the Draft
```
Comment (ajaynegi33) [inline CONTRIBUTING.md:130]: What is this?

**Review by ajaynegi33 — CHANGES_REQUESTED** [review]
Hi @ohmschrodinger,

Thank you for your prompt work on this issue. I appreciate your involvement in the project.

However, after reviewing your contributions, I noticed that you have seemingly copied and pasted content from somewhere else.  I encourage you to focus on making meaningful contributions that add genuine value to the project. It's important to prioritize quality over quantity and avoid making changes that could be seen as spamming the repository.

I appreciate your efforts, and I look forward to seeing more impactful contributions from you in the future! ❤️

**Comment by ohmschrodinger** [comment]
@ajaynegi45 
I understand your concern, but I want to point out that documentation work is about making things clear and easy to understand. It may seem similar to other content because that's how documentation is written to be straightforward.

What I did was to improve the documentation for new contributors, not copy from anywhere. Good documentation is just as important as good code.

**Comment by ajaynegi33** [comment]
> @ajaynegi45 I understand your concern, but I want to point out that documentation work is about making things clear and easy to understand. It may seem similar to other content because that's how documentation is written to be straightforward.
> 
> What I did was to improve the documentation for new contributors, not copy from anywhere. Good documentation is just as important as good code.

The features you mention are not present in the project

**Comment by ohmschrodinger** [comment]
thus the name "contributing"

**Comment by ajaynegi33** [comment]
These features:

Medication Management:**
   * Users can log medications, dosages, and schedules.
   * Reminders for medication times and refill alerts.
4. **Appointment Scheduling:**
   * Feature for users to book, reschedule, and cancel appointments with healthcare providers.
   * Calendar integration to manage and remind users of upcoming appointments.
5. **Health Records Storage:**
   * Secure storage for personal health records and documents.
   * Easy access to past medical history and lab results.
6. **Symptom Checker:**
   * Interactive tool to help users assess symptoms and suggest potential conditions.
   * Recommendations for seeking medical advice based on symptoms.
7. **Health Resources and Articles:**
   * A repository of health-related articles, tips, and guidelines.
   * Curated resources based on user preferences and health interests.
8. **Community Support Forums:**
   * A platform for users to connect with others facing similar health issues.
   * Discussion boards for sharing experiences, advice, and support.
9. **Notifications and Alerts:**
   * Real-time notifications for upcoming appointments, medication reminders, and important health updates.
   * Alerts for any critical health information or changes.
10. **Admin Dashboard:**
    * Features for healthcare providers or administrators to manage users, appointments, and content.
    * Analytics and reporting tools to track user engagement and health outcomes.

**Pull request closed without merging**